pkg/client/add_events_test.go

```diff
@@ -84,7 +84,13 @@ func (s byKey) Swap(i, j int) {
 }
 
 func (s byKey) Less(i, j int) bool {
```

Comment:
Would be good to add a comment / docstring on what's the purpose of this function.

Reply:
I have added comment explaining the code.